doc/apireference.rst

```diff
@@ -120,7 +120,7 @@ File Management and Export
 .. autosummary::
     importers.importStep
     importers.importDXF
-    exporters.exportShape
+    exporters.export
```

Comment:
I expect this change is going to break some existing scripts where users have used `exportShape` directly.

Reply:
I'm not saying "don't do it", I'm just pointing out that we may get issues filed over this change.

Reply:
It is so far only marked as deprecated in the code. I'd propose to remove it in 2.2 so there is enough time for ppl to react on the deprecation warning.

Reply:
Sounds good, +1